src/main/java/frc/robot/subsystems/intake/IntakeIOTalonFX.java

```diff
@@ -0,0 +1,148 @@
+package frc.robot.subsystems.intake;
+
+import com.ctre.phoenix6.configs.CurrentLimitsConfigs;
+import com.ctre.phoenix6.configs.TalonFXConfiguration;
+import com.ctre.phoenix6.controls.DutyCycleOut;
+import com.ctre.phoenix6.controls.TorqueCurrentFOC;
+import com.ctre.phoenix6.hardware.TalonFX;
+import com.ctre.phoenix6.signals.InvertedValue;
+
+import edu.wpi.first.wpilibj.DigitalInput;
+import frc.lib.team6328.util.TunableNumber;
+
+public class IntakeIOTalonFX implements IntakeIO {
+  private TalonFX rightRollerMotor;
+  private TalonFX leftRollerMotor;
+  private TalonFX drumMotor;
+  private final DigitalInput rightRollerIRSensor;
+  private final DigitalInput leftRollerIRSensor;
+  private final DigitalInput drumIRSensor;
```

Comment:
Add instance variables for the signals we need to read:

```
private StatusSignal<Double> leftRollerVelocityStatusSignal;
  private StatusSignal<Double> leftRollerVelocityReferenceStatusSignal;
```

and instance variables for the control requests:

```
private VelocityTorqueCurrentFOC leftRollerVelocityRequest;
```